README.md

```diff
@@ -125,12 +126,13 @@ kubectl create namespace kuadrant
 
 Apply the `Kuadrant` custom resource:
 
-```yaml
-kubectl apply -n kuadrant -f -<<EOF
+```sh
```

Comment:
nitpick: technically it is a shell script, but I would be highlighting the yaml content

Reply:
Plus, if you keep it as `sh`, you can run it in vscode with https://marketplace.visualstudio.com/items?itemName=guicassolato.tothom 😜

Reply:
Yeah, this is a hard one 🤔  TBH previously, we are not consistent in this use for code blocks. In some places, we use `yaml` vs `sh`. 

While highlighting for yaml is nice, I've went with `sh` since it is a shell script and as Gui hinted for vscode, in Goland IDE there is a similar function to run commands directly from markdowns for shell commands also, which is also a nice feature

![image](https://github.com/Kuadrant/kuadrant-operator/assets/24636860/f2d0efd1-c089-49fd-8da2-e1c1e6b67a30)

Reply:
take my money